cli/flags.rs

```diff
@@ -731,10 +744,8 @@ The installation root is determined, in order of precedence:
 These must be added to the path manually if required.")
 }
 
-fn bundle_subcommand<'a, 'b>() -> App<'a, 'b> {
-  SubCommand::with_name("bundle")
-    .arg(lock_arg())
-    .arg(lock_write_arg())
+fn ast_subcommand<'a, 'b>() -> App<'a, 'b> {
+  SubCommand::with_name("ast")
```

Comment:
This is wrong, somehow edited `bundle_subcommand` instead of adding new subcommand

Reply:
Oh, no idea how that happened!

Reply:
Fixed